api-reference/50 Common/Object Structures/positionConfig/of.md

```diff
@@ -4,32 +4,324 @@ type: String | Node | jQuery | Window
 ---
 ---
 ##### shortDescription
-The target element that the widget is positioned against.
+The target element against which the overlay element should be positioned.
```

Comment:
Do you mean:
The target element where an overlay element should be positioned.

Reply:
```suggestion
The target element relative to which the overlay element should be positioned.
```

Reply:
Do you mean:
The target element over which the overlay element is positioned.